Lua/Plugins/py.lua

```diff
@@ -0,0 +1,125 @@
+--- Highligth Keywords
+highlight("import", "reserved")
+highlight("from", "reserved")
+highlight("while", "reserved")
+highlight("if", "reserved")
+highlight("else", "reserved")
+highlight("elif", "reserved")
+
+highlight("False", "binary")
+highlight("True", "binary")
+highlight("None", "binary")
+
+--- Arithmetic Operators
+highlight("+", "operator")
+highlight("-", "operator")
+highlight("*", "operator")
+highlight("/", "operator")
+highlight("%", "operator")
+highlight("**", "operator")
+highlight("//", "operator")
+
+--- Assignment Operators
+highlight("=", "operator")
+highlight("+=", "operator")
+highlight("1=", "operator")
+highlight("*=", "operator")
+highlight("/=", "operator")
+highlight("%=", "operator")
+highlight("//=", "operator")
+highlight("**=", "operator")
+highlight("&=", "operator")
+highlight("|=", "operator")
+highlight("^=", "operator")
+highlight(">>=", "operator")
+highlight("<<=", "operator")
+
+--- Comparison Operators
+highlight("==", "operator")
+highlight("!=", "operator")
+highlight(">", "operator")
+highlight("<", "operator")
+highlight(">=", "operator")
+highlight("<=", "operator")
+
+--- Logical Operators
+highlight("and", "reserved")
+highlight("or", "reserved")
+highlight("not", "reserved")
+
+--- Membership Operators
+highlight("in", "reserved")
+highlight("not in", "reserved")
```

Comment:
There's no need for this one, as the `not` and `in` keywords are already highlighted
```suggestion
```